flybywiresim/a32nx
# feat(efb): add settings page with audio options

## Summary of Changes

- [x] Add PTU in cockpit option
- [x] Add Exterior Master Volume option
- [x] Add Engine Interior Volume option
- [x] Add Wind Interior Volume option
- [x] Persist settings between flights
- [ ] Add numerical indicator next to slider

Discord username (if different from GitHub): someperson#4953

## Testing instructions

* Make sure all four options work in real-time.
* Make sure all four options persist after MSFS restarts (DO NOT use ALT+F4, close MSFS through the main menu)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
